styles/focus/catppuccin.user.css

```diff
@@ -0,0 +1,509 @@
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+    .colored-tables() {
+      table {
+        a {
+          color: @blue;
+        }
+        thead th {
+          background: fade(@accent-color, 30%) !important;
+          color: @text;
+        }
+        tr {
+          &:nth-child(odd) {
+            background: @surface0 !important;
+            a {
+              color: @blue;
+            }
+          }
+          &:nth-child(even) {
+            background: fade(@accent-color, 30%) !important;
+            a {
+              color: @blue;
+            }
+          }
+        }
+      }
+    }
+    .violet {
+      background: @mauve !important;
+    }
+    .purple {
+      background: @lavender !important;
+    }
+    .teal {
+      background: @teal !important;
+    }
+    .olive {
+      background: @yellow !important;
+    }
+    .green {
+      background: @green !important;
+    }
+    .pink {
+      background: @pink !important;
+    }
+    .red {
+      background: @red !important;
+    }
+    .blue {
+      background: @blue !important;
+    }
+    .primary {
+      background: @accent-color !important;
+    }
+    .grey {
+      background: @surface0 !important;
+    }
+
+    .calculated-grade {
+      color: @blue !important;
+    }
+    .button {
+      background: @accent-color;
+      color: @base;
+    }
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+    a {
+      color: @blue;
+    }
+    .swift-box-text-value,
+    .swift-box-button {
+      color: @text !important;
+    }
+    .swift-box-options {
+      border-color: @accent-color;
+    }
+    .swift-box-option-container {
+      color: @text;
+      border-color: @accent-color;
+      background: @overlay0 !important;
+      .swift-box-option-highlight {
+        background: @overlay1;
+        color: @text;
+      }
+      .swift-box-option-filter-container {
+        input {
+          background: @mantle !important;
+          border-color: @crust;
+        }
+        .swift-box-option-filter-icon {
+          color: @text !important;
+        }
+        background: @overlay0;
+        border-color: @accent-color;
+      }
+      .swift-box-option-empty-text {
+        color: @text;
+      }
+    }
+    body,
+    .portal::before,
+    .portal,
+    .content-container,
+    .site-middle,
+    .site-container,
+    .sis-package,
+    .document-ready,
+    .site-content-container,
+    .form {
+      background: @base !important;
+      color: @text;
+    }
+    /* fun hack to get the homepage background to style properly */
+    .portal::before {
+      outline: 30rem solid; /* stylelint-disable-line */
+      outline-color: @base;
+      outline-offset: -20rem !important;
+    }
+    #userwayAccessibilityIcon {
+      img {
+        background-color: @blue !important;
+        border-radius: 50%;
```

Comment:
Why does this need to be set?

Reply:
![Screenshot 2024-08-24 at 10 38 24](https://github.com/user-attachments/assets/ad5e2c20-8f19-4aeb-847f-2516f458a0b4)
this icon

Reply:
What does it look like with the userstyle not applied? Or with the userstyle applied but the border radius commented out?

Reply:
not applied - default blue
![Screenshot 2024-08-24 at 11 14 29](https://github.com/user-attachments/assets/0013f6b3-e8ed-4e60-b9a4-b782cd603597)

border radius removed = weird square border (i could probably use a filter for this tbh)

Reply:
I'm lost on why you need to set the border radius. Try https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-theme-images-and-svgs.

Reply:
Doesn't seem to work for some reason, even directly on the image element

Reply:
Please stop resolving this thread of an ongoing conversation.